### PR TITLE
Select output

### PR DESCRIPTION
**Pull request by pbiology**
## Move selected output to reporting directory

This PR adds a reporting function which takes a subset of files and copies them to a set reporting directory, the idea being that this moves relevant data to a webserver the geneticists can access.

**TODO**
Still there are more files which should be accesible, namely the mapping files (should be in IGV). But this will be taken care of in a separate PR.

### CODE REVIEW

File: tools/helpers.py [hunk 1]
```diff
@@ -277,12 +296,84 @@ def call_script(args):
...
         logger.info(f"Completed the {u.name} pipeline")
+        finished_pipes.append(u.name)
+
+    return finished_pipes
+
+
+def report_results(finished_pipes: list, outdir: str, sample_name: str, config) -> dict:
+    """
+    Takes a list of finished pipelines and checks which output files from this
+    pipeline should be included in the final report (from config file). opies these to the
```
Comment (alvaralmstedt) [inline tools/helpers.py]: Copies*

**Review by alvaralmstedt — APPROVED** [review]
Would be nice to be able to run some easy tests

**Comment by pbiology** [comment]
> Would

Yeah there is a test-stuite on can do. However for this PR I didn't include them

File: tools/helpers.py (hunk 1 above)
Comment (pbiology) [reply tools/helpers.py]: Fixed in e663855

**Pull request merged**